src/public/css/api-documentation.css

```diff
@@ -0,0 +1,85 @@
+body {
+    margin: 0;
+    padding: 0;
+    border:0;
+    font-family: 'Source Sans Pro', Helvetica, Arial, sans-serif;
+}
+
+#ndla_header, #content{
+    background-color: transparent;
+    padding: 20px 0px;
+
+    margin-left: auto;
+    margin-right: auto;
+    min-width: 760px;
+    max-width: 960px;
+}
+
+#content {
+    display: block;
+    clear: both;
+}
+
+a {
+    text-decoration: none;
+    color: #000000;
+}
+
+a:hover {
+    text-decoration: underline;
+}
+
+.home {
+    font-size: xx-large;
+    vertical-align: middle;
+    line-height: 60px;
+}
+
+#slogan {
+    float:right;
+}
+
+#slogan p {
+    width: 220px;
+    margin-left: 10px;
+    float: left;
+    margin-top: 1em;
+    margin-bottom: 1em;
```

Comment:
`margin: 1em 0 1em 10px` (margin: top right bottom left) isteden for margin left, right og top.